benchmarks/tracking_performances/doCompare_truth_real_widebins_mom.C

```diff
@@ -10,7 +10,7 @@
 #define mpi 0.139  // 1.864 GeV/c^2
 
 void draw_req_Mom(double etamin, double etamax, double xmin=0., double xmax=0.);
-void doCompare_truth_real_widebins_mom(TString particle = "pi-",double etamin=-1.0, double etamax=1.0, double range =0.3, Bool_t drawreq=1, TString epic ="", TString eicrecon = "") // name = p, pt for getting p or pt dependence fitted results
+void doCompare_truth_real_widebins_mom(TString particle = "pi-",double etamin=-1.0, double etamax=1.0, double range =0.3, Bool_t drawreq=1, TString extra_legend = "") // name = p, pt for getting p or pt dependence fitted results
```

Comment:
Can I know why you removed the ePIC and EICRecon versions?

Reply:
They are not removed, the whole label is defined in Snakemake. The point is that we only want to specify the campaign tag for official production (the other versions are fixed and known in that case).

Reply:
I see that performances are degraded again. What has changed now? Please check the plot below. If we tag the plot with the version, we will know which version is giving the correct result. It looks again the issue of the material map.
https://eicweb.phy.anl.gov/EIC/benchmarks/detector_benchmarks/-/jobs/3572750/artifacts/file/results/tracking_performances/mom_resol_1.0_eta_2.5.png

Reply:
What you show is not the latest, or not from this PR. The latest job's plot is
https://eicweb.phy.anl.gov/EIC/benchmarks/detector_benchmarks/-/jobs/3571493/artifacts/file/results/tracking_performances/local/mom_resol_1.0_eta_2.5.png

Reply:
Thanks